interfaces/automated-testing/geo-awareness/design/sequence.plantuml

```diff
@@ -0,0 +1,37 @@
+@startuml
+title Geo-Awareness
+
+participant TD as "Test driver"
+participant U as "USS under Test"
+
+== Test setup: Load the Geozones dataset ==
+TD -> U: PUT /geozones/{geozones_id}
+TD <- U: 200 Ok - Geozones loading\n{"result": "Activating"}
+
+TD -> U: GET /geozones/{geozones_id}
+TD <- U: 200 Ok - Geozones still loading\n{"result": "Activating"}
+
+...
+
+TD -> U: GET /geozones/{geozones_id}
+TD <- U: 200 Ok - Geozones ready\n{"result": "Ready"}
+
+== Query the USS for Geo-Awareness information ==
+
+TD -> U: GET /checklist
+TD <- U: 200 Ok - Returns the list of Geo-Awareness information\nfor a specific Volume4D.
```

Comment:
How about "Returns a list of advisories and restrictions (when applicable) for the specified flight"?  (basically, I think a key point here is that the USSP won't simply regurgitate the information they ingested, but rather interpret it as it pertains to the flight and we should expect the resulting actions the USSP should take WRT the operator)

Reply:
Sounds good to me. Thank you for the input.